tests/serde.rs

```diff
@@ -0,0 +1,16 @@
+#![cfg(feature = "serde")]
+#[macro_use]
+extern crate ego_tree;
```

Comment:
Is this still relevant now that this crate is on edition 2021?

Reply:
I refactored all the tests for them to fit the new standard